webapp/tests/karma/ts/selectors/index.spec.ts

```diff
@@ -239,11 +244,8 @@ describe('Selectors', () => {
 
     // null checks
     it('should null check global state', () => {
-      expect(Selectors.getUserFacilityId.projector({})).to.equal(undefined);
-    });
-
-    it('should null check enketo state', () => {
-      expect(Selectors.getEnketoError.projector({})).to.equal(undefined);
+      // @ts-ignore
```

Comment:
This is because, by type definition, it won't be undefined. But we can keep this test to play safe.